rules/prefer-includes.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+const getDocsUrl = require('./utils/get-docs-url');
+
+const isIndexOfCallExpression = node => {
+	if (node.type !== 'CallExpression') {
+		return false;
+	}
+
+	const property = node.callee.property;
+
+	return property.name === 'indexOf';
+};
+
+const isUnaryNotExpression = node => (
+	node.type === 'UnaryExpression' && node.operator === '!'
+);
+
+const isNegativeOne = (operator, value) => operator === '-' && value === 1;
+
+const getSourceCode = (context, node) => (
+	// Context.getSourceCode().text.slice(node.range[0], node.range[1])
+	context.getSourceCode().getText(node)
+);
+
+const report = (context, node, target, pattern) => {
+	const targetSource = getSourceCode(context, target);
+	const patternSource = getSourceCode(context, pattern);
+	context.report({
+		node,
+		message: 'Use .includes(), rather than .indexOf(), when checking for existence.',
+		fix: fixer => fixer.replaceText(node, `${targetSource}.includes(${patternSource})`)
```

Comment:
If we want to support both "includes" and "does not include" cases, this fixer does not cut it. It does not look like it could possibly generate `!a.includes(b)` right now.